apps/state-demo/src/app/examples/demo-basics/2/demo-basics-2.start.component.ts

```diff
@@ -55,7 +55,7 @@ const initComponentState = {
       <button
         mat-raised-button
         color="primary"
-        (click)="onRefreshClicks($event)"
+        (click)="refreshClick.next($event)"
```

Comment:
we have to discuss if subjects should be suffixed with $ or not.

Reply:
Yes good point.

It could be part of the best-practiced document?